packages/cherry-pick-bot/src/bot.ts

```diff
@@ -158,6 +166,21 @@ export = (app: Probot) => {
           ' We cannot authenticate Octokit.'
       );
     }
+
```

Comment:
This handler is only called when a pull request is closed.

We need to add a new handler when a pull request is opened/synchronized/reopened.

Reply:
Added.